src/App.tsx

```diff
@@ -13,20 +11,15 @@ import { HexData } from './Model/hex';
 function App() {
   const [brushSelection, setBrushSelection] = useState({} as BrushSelection);
   const { terrain, icon } = brushSelection;
+  const [hexagons, setHexagons] = useState([] as HexData[])
```

Comment:
nit: useState is a templatable function. Instead of casting `[]`, we can write

`useState<HexData[]>([])`